src/components/common/Actions.vue

```diff
@@ -234,6 +235,9 @@ export default class Actions extends Mixins(AmalgamationMixin, CommonMixin,
       }
     }
 
+    // *** TODO: verify that filing is invalid if the business table now have invalid items
+    // or do we need to await this.$nextTick() here for things to validate?
```

Comment:
I will do this asap.

Reply:
I'm still testing this.

It's the scenario where the primary/holding business' entity type has changed and made the TING table invalid (eg, entity type mismatch).